src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -53,15 +53,15 @@ const AnonymousChannelListScreen = ({route}) => {
     <>
       <StatusBar translucent={false} />
       <View style={{height: 52}}>
-        <Search route={route} onPress={goToContactScreen} isShowNewChat={false} />
+        <Search route={route} isAnon={true} onPress={() => goToContactScreen({from: ANONYMOUS})} />
       </View>
 
       <FlatList
         data={anonChannels}
         keyExtractor={(item) => item.id}
         ListHeaderComponent={
           <ChannelListHeaderItem
-            name="Anonymous"
+            name="Incognito"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `name` prop of `ChannelListHeaderItem` has been changed from "Anonymous" to "Incognito". This change is likely for user interface purposes and doesn't seem to affect the functionality of the code.
```diff
- name="Anonymous"
+ name="Incognito"
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->